src/mapping.ts

```diff
@@ -77,8 +85,12 @@ export function handleTrade(event: Trade): void {
       buyToken.priceEth = buyTokenPrices.get("eth")
     }
   } else {
+    let buyTokenAddressHexString = buyTokenAddress.toHexString()
+    if (buyTokenAddressHexString === "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"){
```

Comment:
Shouldn't you refactor these 2 lines? you do then in both sides of the IF/ELSE
maybe just declare a `isBuyETH` flag?